src/components/SelectDropdown/index.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React, { Component } from 'react'
+import Dropdown from '@components/Dropdown'
+import DropdownTrigger from '@components/DropdownTrigger'
+import Icon from '@components/Icon'
+import Text from '@components/Text'
+import Button from '@components/Button'
+import {
+  Wrapper,
+  DropdownBubble,
+  OptionsList,
+  OptionRow,
+  DropdownActions
+} from './styles'
+
+type Props = {
+  onApply: (selectedOptions: string[]) => {},
+  options: { value: string, text: string }[]
+}
+
+type State = {
+  selectedOptions: string[]
+}
+
+class SelectDropdown extends Component<Props, State> {
+  state = {
+    selectedOptions: []
+  }
+
+  selectItem = (value: string) => {
+    const updatedSelectedOptions = [...this.state.selectedOptions]
+    const index = updatedSelectedOptions.findIndex(opt => opt === value)
```

Comment:
This is basically `updatedSelectedOptions.indexOf(value)`, right? Using `findIndex` is a bit overkill here.